app/Http/Controllers/Admin/MonsterCrudController.php

```diff
@@ -240,76 +337,97 @@ public function setupShowOperation()
             'type'          => 'model_function_attribute',
             'function_name' => 'getCategory', // the method in your Model
             'attribute'     => 'name',
+            'tab'           => 'Function/Closure',
```

Comment:
I think we can move this to miscellaneous or something similar, no need for a tab for this function/closure thing.

Reply:
Done